gallery/sbol_visual/gene_cassette.py

```diff
@@ -0,0 +1,39 @@
+import sbol3 as sbol
```

Comment:
I added this script to (re)generate the SBOLv3 xml for the Gene Cassete example design